test/e2e/fixtures/directive-require-html/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html ng-app="test" require-directive parent-directive>
```

Comment:
`parentDirective` isn't a parent directive. I would name it `siblingDirective` or `notParentDirective` to make it obvious.
(I was kind of confused at first.)

Reply:
Well it's supposed to be the parent Directive. The require directive requires it on one of its parent elements but also has it itself for the sake of the test.

Reply:
No, it's a bug in `requireDirective` 😛

Reply:
No, the user uses it incorrectrly! :D